dask_cuda/tests/test_dask_cuda_worker.py

```diff
@@ -42,7 +45,49 @@ def get_visible_devices():
 
                     # verify 4 workers with the 4 expected CUDA_VISIBLE_DEVICES
                     result = client.run(get_visible_devices)
-                    expected = {"2,3,7,8": 1, "3,7,8,2": 1, "7,8,2,3": 1, "8,2,3,7": 1}
+                    expected = {"0": 1}
+                    for v in result.values():
+                        del expected[v]
+                    assert len(expected) == 0
+    finally:
+        del os.environ["CUDA_VISIBLE_DEVICES"]
+
+
+def test_cuda_visible_devices(loop):
+    n_gpus = get_n_gpus()
+    if n_gpus < 2:
+        pytest.skip("More than 1 GPU required for test")
+    os.environ["CUDA_VISIBLE_DEVICES"] = cuda_visible_devices(0)
```

Comment:
Same as above.  This test can probably be made to work without the presense of GPUs.